Client/Assets/About/Licenses.html

```diff
@@ -454,33 +454,6 @@ <h4 id="exhibit-b---incompatible-with-secondary-licenses-notice">Exhibit B - “
 <center><p>-</p></center>
 
```

Comment:
The changes here looks funky.  But what happen was that I had not removed the licence for SWTableViewCell so I removed that as well along with SWXMLHash. Then added Fuzi